dev/benchmarks/macrobenchmarks/lib/src/filtered_child_animation.dart

```diff
@@ -27,25 +27,21 @@ class FilteredChildAnimationPage extends StatefulWidget {
 }
 
 class _FilteredChildAnimationPageState extends State<FilteredChildAnimationPage> with SingleTickerProviderStateMixin {
-  AnimationController _controller;
+  late final AnimationController _controller = AnimationController(vsync: this, duration: const Duration(seconds: 2));
   final GlobalKey _childKey = GlobalKey(debugLabel: 'child to animate');
   Offset _childCenter = Offset.zero;
 
-  FilterType _filterType;
-  bool _complexChild;
-  bool _useRepaintBoundary;
+  late FilterType? _filterType = widget.initialFilterType;
```

Comment:
`late FilterType?`?

Reply:
Because 
```
void _setFilterType(FilterType type, bool selected) {
    setState(() => _filterType = selected ? type : null);
  }
```

This function can assign FilterType to null

Reply:
I see, you want it lazily instantiated, but it's nullable.

Reply:
yes exactly.

Reply:
2000umairkhan 5673568467 0503 2564 05